src/components/Nursery/index.tsx

```diff
@@ -102,6 +107,16 @@ export default function NurseryDetails(): JSX.Element {
             display={true}
           />
         </Grid>
+        {timeZoneFeatureEnabled && (
+          <Grid item xs={gridSize()}>
```

Comment:
it will be cool to inform user if the tz is being inherited from org.

Reply:
I thought we could add that later as an info tooltip in the label (autocomplete does not support tooltip yet). Thoughts?

Reply:
Oh yes, tooltip works great.